Source/CarthageKit/Project.swift

```diff
@@ -1282,23 +1283,33 @@ internal func frameworksInDirectory(_ directoryURL: URL) -> SignalProducer<URL,
 	return filesInDirectory(directoryURL, kUTTypeFramework as String)
 		.filter { !$0.pathComponents.contains("__MACOSX") }
 		.filter { url in
-
 			// Skip nested frameworks
 			let frameworksInURL = url.pathComponents.filter { pathComponent in
 				return (pathComponent as NSString).pathExtension == "framework"
 			}
 			return frameworksInURL.count == 1
 		}.filter { url in
-
-			let packageType: PackageType? = Bundle(url: url)?.packageType
+			// Try reading the Info.plist of the Framework
```

Comment:
I’m confident in the structure of the CLI output from `objdump` rarely changing (and if it does, we should be able to catch via open-sourcedness before it’s widely distributed), but a simple Plist read is probably faster than a process execution. If you agree, could we leave this the default and add a comment with the reasoning?

```
// For reasons of speed and the fact that CLI-output structures can change,
// first try the safer method of reading the ‘Info.plist’ from the Framework’s bundle.
```

Reply:
Sure